src/makeCalculator.js

```diff
@@ -36,8 +36,47 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, value) {
+      return callback.call(this, value);
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(value) {
+      this.result += value;
+
+      return this;
+    },
```

Comment:
If you do as in the first comment, this code can be shortened and your code style will be cleaner. Fix this in all methods

```suggestion
    add(value) {
      this.result += value;
    },
```